lib/tasks/temp.rake

```diff
@@ -1,6 +1,14 @@
 # -*- coding: utf-8 -*-
 namespace :temp do
 
+  desc 'Populate last_event_time column of InfoRequest'
+  task :populate_last_event_time => :environment do
+    InfoRequest.where('last_event_time is NULL').find_each do |info_request|
```

Comment:
Also - would it speed this up at all if we eager loaded the info_request_events?